config/locales/ru.yml

```diff
@@ -0,0 +1,269 @@
+---
+ru:
+  activerecord:
+    errors:
+      messages:
+        record_invalid: 'Возникли ошибки: %{errors}'
+        restrict_dependent_destroy:
+          has_one: 'Невозможно удалить запись, так как существует зависимость: %{record}'
+          has_many: 'Невозможно удалить запись, так как существуют зависимости: %{record}'
+  date:
+    abbr_day_names:
+    - Вс
+    - Пн
+    - Вт
+    - Ср
+    - Чт
+    - Пт
+    - Сб
+    abbr_month_names:
+    -
+    - янв.
+    - февр.
+    - марта
+    - апр.
+    - мая
+    - июня
+    - июля
+    - авг.
+    - сент.
+    - окт.
+    - нояб.
+    - дек.
+    day_names:
+    - воскресенье
+    - понедельник
+    - вторник
+    - среда
+    - четверг
+    - пятница
+    - суббота
+    formats:
+      default: "%d.%m.%Y"
+      long: "%-d %B %Y"
+      short: "%-d %b"
+    month_names:
```

Comment:
Just specifying in case this matters - these month names are spelled to be used with a day.. like **9 января**. The month not used with this day context, for ex. "next month is January" would be **январь** 

This is the case for all of these months, from the `formats` above it looks like they are meant to be used with a day so that would be correct, but noting the other ending of the word in case it is useful 

(Sorry I do not use specific grammar-related nomenclature in my explanation, not a grammar scholar)

Reply:
thanks for pointing. I think, we are using month with the day(in showing dates) and as the translation system doesn't work for research note body, etc. so I think, we will be good here.